src/pupil_labs/invisible_lsl_relay/pi_relay.py

```diff
@@ -22,8 +22,8 @@ def __init__(self, channel_func, outlet_name, outlet_format, timestamp_query,
 
     def push_sample_to_outlet(self, sample):
         try:
-            sample_to_push = [chan.query(sample) for chan in self._channels]
-            timestamp_to_push = self._timestamp_query(sample)
+            sample_to_push = [chan.sample_query(sample) for chan in self._channels]
+            timestamp_to_push = self._timestamp_query(sample) - self._time_offset
```

Comment:
self._time_offset is the difference between a timestamp made with time.time() at the moment when an Instance of the PupilInvisible Relay class is created and the lsl clock.

However, the timestamp we are pushing here was originally collected on a different system (companion device) then the system on which this code will run. Can we assume that time.time on the device where this code runs will be the same as time.time on our companion device? 
It looks like different systems might use different epochs, which is worrying to me:
https://docs.python.org/3/library/time.html#time.time